### PR TITLE
Update validator with the latest spec changes

### PR DESCRIPTION
**Pull request by Jesus89**
- Add "validator-examples" job in CI
- Use "version" instead of "schema_version"
- Add "geometry_type"
- Add "epoch"
- Move "crs" optional

The example has been updated.

### CODE REVIEW

**Review by kylebarron — APPROVED** [review]

File: .github/workflows/scripts.yml [hunk 1]
```diff
@@ -7,6 +7,28 @@ on:
...
+      - name: Install validator
+        run: |
+          cd validator/python
+          python -m pip install --no-binary geoparquet_validator .
+
+      - name: Run validator
+        run: |
+          for example in $(ls examples/*.parquet); do
+            echo $example;
+            geoparquet_validator $example || exit 1;
```
[the remark this answers is not in the document]
Comment (Jesus89) [reply .github/workflows/scripts.yml:29]: I tested in a separated bash file, with an invalid GeoParquet file, and didn't work (stop on the fail) until I added explicit exit 1. But we can test it later and remove it if it's not necessary.

File: examples/example.py [hunk 2]
```diff
@@ -39,7 +39,7 @@
 HERE = pathlib.Path(__file__).parent
```
[the remark this answers is not in the document]
Comment (Jesus89) [reply examples/example.py:39]: Sure

File: format-specs/geoparquet.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ Each geometry column in the dataset must be included in the columns field above
 
 #### crs
 
-The Coordinate Reference System (CRS) is an optional parameter for each geometry column defined in geoparquet format. 
+The Coordinate Reference System (CRS) is an optional parameter for each geometry column defined in geoparquet format.
```
[the remark this answers is not in the document]
Comment (Jesus89) [reply format-specs/geoparquet.md:65]: Great idea :+1:

File: .github/workflows/scripts.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kylebarron) [reply .github/workflows/scripts.yml:29]: Did you test locally or on CI? Locally you should be able to see the exit code with `echo $?` after a command. On CI I believe github actions checks the return code after each bash command.

**Pull request merged**